openai.el

```diff
@@ -69,6 +69,10 @@ The function should take no arguments and return a string containing the key.
 A function, `openai-key-auth-source', that retrieves the key from
 auth-source is provided for convenience.")
 
+(defvar openai-key-type :bearer
+  "Variable storing the openai key type.
+Should be one of either :bearer or :azure-api.")
```

Comment:
We can use `defcustom` instead since we want users to customize this variable.

Reference: https://github.com/emacs-vs/line-reminder/blob/d8437c33154b25d0499cb8929409ed438486e7b9/line-reminder.el#L47-L51

Reply:
makes sense!